<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MosaicDialog</class>
 <widget class="QDialog" name="MosaicDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="mosaic_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="mosaic_spinBox">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>281</y>
     <width>46</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSlider" name="mosaic_slider">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>285</y>
     <width>84</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="mosaic_ok">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>361</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mosaic_cancel">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>361</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
